stories/04-wordpress-components/image.stories.mdx

```diff
@@ -1 +1,20 @@
 <Meta title="WordPress Components|Image" />
+
+# Image
+Output an image.
+
+Base image component supporting various display options including alt, src, srcset, lazy loding, fallback, captions, credits, and more.
+
```

Comment:
would be nice to see an example that uses at least two options, one of which is the alt text so we are modeling good accessibility practices. an example showing a more complex option would be helpful